openpaas-james/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/openpaas/james/common/LinagoraKeystoreSetMethodContract.scala

```diff
@@ -33,10 +81,10 @@ trait LinagoraKeystoreSetMethodContract {
                      |	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:pgp"],
                      |	"methodCalls": [
                      |		["Keystore/set", {
-                     |			"accountId": "$generateAccountIdAsString",
+                     |			"accountId": "$ACCOUNT_ID",
                      |			"create": {
-                     |        "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6": {
-                     |          "key": "GPG Armored"
+                     |        "K87": {
+                     |          "key": "$PGP_KEY_BASE_64"
```

Comment:
In base64 because well you can really pass an array of bytes in an other way as json it seems...

Reply:
Why do we need this conversion:

```
armored key (text) -> bytes -> base64 (text) ?
```

Can't we have the raw armored key text passed here?

Reply:
Then let me know how to deal with that with play json and deserialize it cause that caused some json parse exception to me and I could not find a workaround

Reply:
> Then let me know how to deal with that with play json and deserialize it cause that caused some json parse exception to me and I could not find a workaround

Technical constraint is a bad reason to design a bad API, Rene.

A String is a string be it base64 or armored. I suspect you rather have problems with scala interpolation and line breaks.

Reply:
I guess it's probably better towards what we want but saying it would be a bad API... I don't see how. 

Anyway, I spent a good amount of time on that already and can't make it work that way, json serialization is driving me nuts. So support on this would be appreciated.